src/components/Form/Form.scss

```diff
@@ -0,0 +1,147 @@
+@use "../../styles/base/typography" as *;
+
+@media screen and (min-width: $mobile-breakpoint) {
+  .form-container {
+    padding: 20px 20px;
+    height: fit-content;
+    width: 100%;
+    box-shadow: 0px 20px 40px rgba(114, 75, 203, 0.2);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 10px;
+
+    &__category {
+      grid-area: category;
+      width: 100%;
+    }
+    &__staff {
+      grid-area: staff;
+      width: 100%;
+    }
+    &__resource {
+      grid-area: resource;
+      width: 100%;
+    }
+    &__checkbox {
+      grid-area: checkbox;
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: center;
+      color: $page-sub-heading;
+      width: 100%;
+
+      &--label {
+        font-family: $font-modern;
+        font-weight: 600;
+        font-size: 0.8rem;
+        color: $page-sub-heading;
+      }
+
+      &--yes,
+      &--no {
+        display: flex;
+        gap: 10px;
+        font-size: 0.8rem;
+      }
+    }
+    &__checkboxes {
+      display: flex;
+      flex-direction: row;
+      width: 100%;
+      gap: 50px;
+      width: 100%;
+    }
+    &__auto-purchase {
+      grid-area: auto-purchase;
+      width: 100%;
+    }
+    &__quantity-remaining {
+      grid-area: quantity-remaining;
+      width: 100%;
+    }
+  }
+
+  label {
+    color: $page-sub-heading;
+    font-family: $font-modern;
+    font-size: 0.8rem;
+  }
+  .input-field {
+    width: 100%;
+
+    &__label {
+      color: $page-sub-heading;
+      font-family: $font-modern;
+    }
+    &__input {
+      width: auto;
+    }
+  }
+  .dropdown-field {
+    width: 100%;
+    &__label {
+      color: $page-sub-heading;
+      font-family: $font-modern;
+    }
+  }
+}
+
+@media screen and (min-width: $tablet-breakpoint) {
+  .form-container {
+    gap: 50px;
+    padding: 10px 20px;
+    height: fit-content;
+    width: 80%;
+    box-shadow: 0px 20px 40px rgba(114, 75, 203, 0.2);
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    gap: 0px 30px;
+    grid-template-areas:
+      "category checkbox"
+      "staff auto-purchase"
+      "resource quantity-remaining";
+
+    &__category {
+      grid-area: category;
+      padding: 10px;
+    }
+    &__staff {
+      grid-area: staff;
+      padding: 10px;
+    }
+    &__resource {
+      grid-area: resource;
+      padding: 10px;
+    }
+    &__checkbox {
+      grid-area: checkbox;
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: center;
+      color: $page-sub-heading;
+      padding: 10px;
+    }
+    &__checkboxes {
+      display: flex;
+      flex-direction: row;
+      width: 100%;
+      gap: 50px;
+    }
+    &__auto-purchase {
+      grid-area: auto-purchase;
+      padding: 10px;
+    }
+    &__quantity-remaining {
+      grid-area: quantity-remaining;
+      padding: 10px;
+    }
+  }
+  .button {
```

Comment:
Never call _.button_ as a styling convention as it restyles every appearance of a button. put your buttons in a named html container and style that.